addons/website/views/snippets/s_accordion_image.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_accordion_image" name="Accordion Image">
+    <section class="s_accordion_image pt56 pb56 o_cc o_cc2">
+        <div class="container">
+            <div class="row align-items-start">
+                <div class="col-lg-7 pt16">
+                    <h2 class="h3-fs">Top questions answered</h2>
+                    <p class="lead">In this section, you can address common questions efficiently.</p>
+                    <p><br/></p>
+                    <t t-call="website.s_accordion">
+                        <t t-set="extra_classes" t-valuef="accordion-flush"/>
+                    </t>
+                </div>
+                <div class="col-lg-5 pt16" data-name="Accordion Box">
```

Comment:
Not sure `data-name="Accordion Box"` is useful. I think that keeping default "Column" will do the job ?

Reply:
This was made to maintain consistency with `s_faq_collapse`:) 
<img width="1792" alt="Capture d’écran 2024-08-20 à 11 21 02" src="https://github.com/user-attachments/assets/f78c5c2b-b11a-4d31-ae6f-d624efcbc59f">

Reply:
Okay that's what I thought, it was not intended... You set it on the image column ;)

Reply:
Oops good catch ! Fixed ✅